{{cookiecutter.project_shortname}}/scripts/setup

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+set -e
+
+if [ "$1" == "--dev" ]; then
+    export INVENIO_SQLALCHEMY_DATABASE_URI="postgresql+psycopg2://{{cookiecutter.project_shortname}}:{{cookiecutter.project_shortname}}@localhost/{{cookiecutter.project_shortname}}"
+fi
+
+# Clean redis
+pipenv run invenio shell --no-term-title -c "import redis; redis.StrictRedis.from_url(app.config['CACHE_REDIS_URL']).flushall(); print('Cache cleared')"
+pipenv run invenio db destroy --yes-i-know
+pipenv run invenio db init create
+pipenv run invenio index destroy --force --yes-i-know
+pipenv run invenio index init --force
+pipenv run invenio index queue init purge
+pipenv run invenio files location --default 'default-location'  $(pipenv run invenio shell --no-term-title -c "print(app.instance_path)")
+
+# Create admin role to rectict access
+pipenv run invenio roles create admin
+pipenv run invenio access allow superuser-access role admin
```

Comment:
normal: remove all the `pipenv` and just call this script using `pipenv` .

In our implementation, we also removed all the destructive commands. It is easy to mistakenly call `setup` a second time and then very '#sad' to see it blast the DB away.

Reply:
Same as above on scripts.

Reply:
sure